h/util/db.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import sqlalchemy
+
+
+def on_transaction_end(session):
+    """
+    Decorator for a function which should run after a top-level transaction ended.
+
+    Transaction that are either implicitly or explicity committed or rolled back will be
+    closed at the end of a Pyramid view. This is here for cleaning up caches so that
+    code after the view, exception views for example, will not be able to access
+    detached instances.
+
+    The wrapped function will receive the database session as ``session`` and the
```

Comment:
Is there any reason to pass these through at the moment? I think I'd lean towards not doing so unless there's a need, as it will at least prevent unused argument warnings.